.travis.yml

```diff
@@ -70,7 +70,7 @@ install:
       gettext git-core gperf libasound2-dev libass-dev libbluray-dev libbz2-dev libcap-dev libcdio-dev libcec4-dev libcrossguid-dev libcurl3
       libcurl4-openssl-dev libdbus-1-dev libegl1-mesa-dev libfmt3-dev libfontconfig-dev libfreetype6-dev libfribidi-dev libfstrcmp-dev libgif-dev libgl1-mesa-dev
```

Comment:
```suggestion
      libcurl4-openssl-dev libdbus-1-dev libegl1-mesa-dev libfmt-dev libfontconfig-dev libfreetype6-dev libfribidi-dev libfstrcmp-dev libgif-dev libgl1-mesa-dev
```

Reply:
do we even still use travis? If yes, the nightly PPA would need to be added for libfmt and libspdlog dependecies

Reply:
I have no idea but if we do it was probably already broken... Should I fix this anyway?

Reply:
Imho it's out of scope for this PR. Let's check if this is still used and if needed fix it in a separate PR